services/state/token/account.proto

```diff
@@ -97,11 +97,11 @@ message Account {
      */
     int64 head_token_number = 13;
     /**
-     * The NftId of the head of the linked list from unique tokens map for the account.
+     * The NftID of the head of the linked list from unique tokens map for the account.
      */
     int64 head_nft_id = 14;
```

Comment:
Should this be NftID head_nft_id=14?

Reply:
That change will be made as part of issue #7243

Reply:
I see. Thanks.